test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -341,6 +341,37 @@ def test_wire_order(self):
             str(_text_circuit_drawer(circuit, wire_order=[2, 1, 3, 0, 6, 8, 9, 5, 4, 7])), expected
         )
 
+    def test_wire_order_only_qubits(self):
+        """Test the wire_order option with only qubits"""
+        expected = "\n".join(
+            [
+                "                    ",
+                "q_2: |0>────────────",
+                "        ┌───┐       ",
+                "q_1: |0>┤ X ├───────",
+                "        ├───┤ ┌───┐ ",
+                "q_3: |0>┤ H ├─┤ X ├─",
+                "        ├───┤ └─╥─┘ ",
+                "q_0: |0>┤ H ├───╫───",
+                "        └───┘┌──╨──┐",
+                " c: 0 4/═════╡ 0xa ╞",
+                "             └─────┘",
+                "ca: 0 2/════════════",
+                "                    ",
+            ]
+        )
+        qr = QuantumRegister(4, "q")
+        cr = ClassicalRegister(4, "c")
+        cr2 = ClassicalRegister(2, "ca")
+        circuit = QuantumCircuit(qr, cr, cr2)
+        circuit.h(0)
+        circuit.h(3)
+        circuit.x(1)
+        circuit.x(3).c_if(cr, 10)
+        self.assertEqual(
+            str(_text_circuit_drawer(circuit, wire_order=[2, 1, 3, 0, 4, 5, 6, 7, 8, 9])), expected
+        )
```

Comment:
This test says its `wire_order` is only qubits, but it actually has all bits.

Reply:
test removed. See https://github.com/Qiskit/qiskit-terra/pull/9893#discussion_r1262414808